tgui/packages/tgui/interfaces/Fabricator.tsx

```diff
@@ -134,23 +134,23 @@ export const Fabricator = (props, context) => {
   const categoryCounts: { [K: string]: number } = {};
   let totalRecipes = 0;
 
-  for (let design of sortedDesigns) {
+  for (const design of sortedDesigns) {
     totalRecipes += 1;
```

Comment:
this could possibly be simplified to `const totalRecipes = Object.keys(sortedDesigns).length`

Reply:
Yeah, but there's also the inner loop. I'm not sure if this would make it any more readable...